go-controller/pkg/ovn/policy.go

```diff
@@ -1492,3 +1495,80 @@ func (oc *Controller) shutdownHandlers(np *networkPolicy) {
 		oc.watchFactory.RemoveServiceHandler(handler)
 	}
 }
+
+// iterateRetryNetworkPolicies checks if any outstanding NetworkPolicies exist
+// then tries to re-add them if so
+// updateAll forces all policies to be attempted to be retried regardless
+func (oc *Controller) iterateRetryNetworkPolicies(updateAll bool) {
+	oc.retryNetPolices.retryMutex.Lock()
+	defer oc.retryNetPolices.retryMutex.Unlock()
+	now := time.Now()
+	for namespacedName, npEntry := range oc.retryNetPolices.entries {
+		if npEntry.ignore {
+			continue
+		}
+		// check if we need to create
+		var policyToCreate *knet.NetworkPolicy
+		if npEntry.newObj != nil {
+			p := npEntry.newObj.(*knet.NetworkPolicy)
+			// get the latest version of the new policy from the informer, if it doesn't exist we are not going to
+			// create the new policy
+			np, err := oc.watchFactory.GetNetworkPolicy(p.Namespace, p.Name)
+			if err != nil && kerrors.IsNotFound(err) {
+				klog.Infof("%s policy not found in the informers cache, not going to retry policy create", namespacedName)
+				npEntry.newObj = nil
+			} else {
+				policyToCreate = np
+			}
+		}
+
+		npEntry.backoffSec = npEntry.backoffSec * 2
+		if npEntry.backoffSec > 60 {
+			npEntry.backoffSec = 60
+		}
+		backoff := (npEntry.backoffSec * time.Second) + (time.Duration(rand.Intn(500)) * time.Millisecond)
+		npTimer := npEntry.timeStamp.Add(backoff)
+		if updateAll || now.After(npTimer) {
```

Comment:
NIT: It may save us the long indented block if we rewrote this block as:
```
if !updateAll && now.Before(npTimer) {
    klog.V(5).Infof("%s retry network policy not after timer yet, time: %s", namespacedName, npTimer)
    continue
}

... <the original block, now not needed to be inside long "if { }" block
```

Reply:
original code not me :)